src/frontend/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import "./Sidebar.css"
+
+const Sidebar = () => {
+
+    const optionSelected = ({ isActive }) => {
+        return "sidebar__option" + (isActive ? "--selected" : "") + " flex--row";
```

Comment:
Suggestion - You could achieve this by using a callback passed to style={}.

Reply:
thought using a class rather than inline style was preferred ..